From: Lucas Stach <l.stach@pengutronix.de>
Date: Tue, 17 Oct 2017 12:29:22 +0200
Subject: [PATCH] add largefile support

Signed-off-by: Lucas Stach <l.stach@pengutronix.de>
---
 configure.ac | 5 +++++
 1 file changed, 5 insertions(+)

diff --git a/configure.ac b/configure.ac
index 119ef6005103..e9b4ebe32c03 100644
--- a/configure.ac
+++ b/configure.ac
@@ -38,6 +38,9 @@ AM_PROG_LIBTOOL
 
 CC_NOUNDEFINED
 
+dnl enable largefile
+AC_SYS_LARGEFILE
+
 dnl Checks for header files.
 AC_HEADER_STDC
 AC_CONFIG_HEADERS(include/config.h)
@@ -706,6 +709,8 @@ if test ! -L "$srcdir"/include/alsa ; then
   ln -sf . "$srcdir"/include/alsa
 fi
 
+CPPFLAGS='-include $(CONFIG_HEADER)'
+
 AC_OUTPUT(Makefile doc/Makefile doc/pictures/Makefile doc/doxygen.cfg \
 	  include/Makefile include/sound/Makefile src/Versions src/Makefile \
           src/control/Makefile src/mixer/Makefile \
